docs/openstack-compute-kit-neutron.md

```diff
@@ -96,3 +96,39 @@ sed 's/your.domain.tld/<YOUR_DOMAIN>/g' \
 ``` shell
 kubectl --namespace openstack apply -f /etc/genestack/gateway-api/routes/custom-neutron-gateway-route.yaml
 ```
+
+## Neutron MTU settings / Jumbo frames / overlay networks on instances
+
+!!! warning You will likely need to increase the MTU as described here if you want to support creating L3 overlay networks (via any software that creates nested networks, such as _Genestack_ itself, VPN, etc.) on your nova instances. Your physical L2 network will need jumbo frames to support this. You will likely end up with a an MTU of 1280 for overlay networks on instances if you don't, and the abnormally small MTU can cause various problems, perhaps even reaching a size too small for the software to support).
+
+You might want to reference [Neutron documentation on MTU considerations](https://docs.openstack.org/neutron/latest/admin/config-mtu.html)
```

Comment:
perhaps remove the ambiguity of "you might want to" and just use the link

Reply:
sounds good. changed